src/main/java/uk/gov/hmcts/reform/dg/docassembly/Application.java

```diff
@@ -7,7 +7,12 @@
 import org.springframework.context.annotation.EnableAspectJAutoProxy;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
-@SpringBootApplication(exclude = { DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class })
+@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class},
+    scanBasePackages = {
+        "uk.gov.hmcts.reform.ccd.document.am",
+        "uk.gov.hmcts.reform.dg.docassembly",
+        "uk.gov.hmcts.reform.auth"}
+)
 @EnableScheduling
```

Comment:
I think we don't need this any more. Can we remove this? And also @EnableAspectJAutoProxy(proxyTargetClass = true) ?
Also, can we remove and check if exclude = {DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class} is still required?

Reply:
removed